www_src/lib/panzoom-jquery.js

```diff
@@ -0,0 +1,1199 @@
+/* jshint ignore:start */
```

Comment:
All we really need are the matrix utilities and the stuff in `_startMove`, but there needs to be some work done to decouple options, jquery, etc.